baseapp/baseapp_test.go

```diff
@@ -717,10 +717,10 @@ func (msg msgCounter) String() string { return "TODO" }
 func (msg msgCounter) ProtoMessage()  {}
 
```

Comment:
could you also remove the 9418 entry in the CHANGELOG?

Reply:
[removed](https://github.com/cosmos/cosmos-sdk/pull/9915/commits/6eece0b20e5a929bea2ef5b2c977088ed4261ccc#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL62)